ale/drivers/selene_drivers.py

```diff
@@ -278,7 +295,7 @@ def ephemeris_start_time(self):
         : float
           ephemeris start time of the image
         """
-        return spice.sct2e(self.spacecraft_id, self.spacecraft_clock_start_count)
+        return spice.sct2e(self.spacecraft_id, float(self.spacecraft_clock_start_count))
```

Comment:
Does it make sense to cast here or should the spacecraft_clock_start_count always be a float? So, cast in that property.

Reply:
Space craft clock counts are actually strings most of the time:

https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/req/time.html#Spacecraft%20Clock%20SCLK

What's happening here is that we're calling sct2e which takes space craft clock ***ticks***. There is some inconsistency in labels between the spacecraft clock counts being strings or floats. If it's a string we should be using [scs2e](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/scs2e_c.html) instead of [sct2e](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/sct2e_c.html).